docs/changelogs/v0.26.md

```diff
@@ -30,6 +31,10 @@ Several deprecated commands have been removed:
 - `ipfs dns` deprecated in [April 2022, Kubo 0.13](https://github.com/ipfs/kubo/commit/76ae33a9f3f9abd166d1f6f23d6a8a0511510e3c), use `ipfs resolve /ipns/{name}` instead.
 - `ipfs tar` deprecated [April 2022, Kubo 0.13](https://github.com/ipfs/kubo/pull/8849)
 
+#### Support optional pin names
+
+You can now add a name to a pin when pinning a CID. To do so, use `ipfs pin add --name "Some Name" bafy...`. You can list your pins, including their names, with `ipfs pin ls --detailed`.
```

Comment:
:thought_balloon: @hacdias  maybe keep this simple and rename to `pin ls` parameter to `-n`  `--names`? Feels like better UX, and it is unlikely we will have any additional metadata than names any time soon, and if we ever add it, that will be a bigger refactor and another param.